examples/image_classifier/README.md

```diff
@@ -0,0 +1,25 @@
+# Example: Image Classifier
```

Comment:
Metaflow flows canonically end with `_flow` to distinguish them from model classes and supporting libraries. So in this case it would be `image_classifier_flow.py`.